### PR TITLE
Head idempotent: Run 'errata-cache' to get errata info updated after installing pkgs

### PR DESCRIPTION
**Pull request by meaksh**

### CODE REVIEW

File: features/trad_need_reboot.feature [hunk 1]
```diff
@@ -21,16 +21,25 @@ Feature: Reboot required after patch
...
   Scenario: enable old-packages for test a "needing reboot"
+    Given I am authorized as "admin" with password "admin"
     And I run "zypper -n mr -e Devel_Galaxy_BuildRepo" on "sle-client"
+    And I run "zypper -n ref" on "sle-client"
     And I run "zypper -n in --oldpackage andromeda-dummy-1.0-4.1" on "sle-client"
     And I run "rhn_check -vvv" on "sle-client"
-    And I run "zypper -n ref" on "sle-client"
+    When I follow "Admin"
+    And I follow "Task Schedules"
+    And I follow "Task Schedules"
```
Comment (MalloZup) [inline features/trad_need_reboot.feature]: NICE! TASKO GO TASKO

**Comment by MalloZup** [comment]
ok, we need to rename the feature

File: features/trad_check_patches_install.feature [hunk 2]
```diff
@@ -6,6 +6,21 @@ Feature: Check patches
...
+    And I run "zypper -n in --oldpackage andromeda-dummy-1.0-4.1" on "sle-client"
+    And I run "rhn_check -vvv" on "sle-client"
+    When I follow "Admin"
+    And I follow "Task Schedules"
+    And I follow "Task Schedules"
+    And I follow "errata-cache-default"
+    And I follow "errata-cache-bunch"
+    And I click on "Single Run Schedule"
+    Then I should see a "bunch was scheduled" text
+    And I wait for "5" seconds
```
Comment (MalloZup) [inline features/trad_check_patches_install.feature]: here we need to find a way to wait din

File: features/core_min_salt_ssh.feature [hunk 3]
```diff
@@ -38,6 +38,13 @@ Feature: 1) Bootstrap a new salt host via salt-ssh
     And I click on "Modify Base Software Channel"
     And I should see a "System's Base Channel has been updated." text
 
+  Scenario: Apply the highstate on ssh-minion to apply base channels
+     Given I am on the Systems overview page of this "ssh-minion"
+     And I follow "States" in the content area
+     And I click on "Apply Highstate"
+     Then I should see a "Applying the highstate has been scheduled." text
+     And I wait for "40" seconds
```
Comment (MalloZup) [inline features/core_min_salt_ssh.feature]: here too

**Comment by meaksh** [comment]
@MalloZup - I merged since your comments are fixed. Let's trigger a testsuite run! 🚀 🚀

**Pull request merged**